app/renderer/js/notification/darwin-notifications.ts

```diff
@@ -39,45 +44,45 @@ class DarwinNotification {
 		notification.addEventListener('reply', this.notificationHandler);
 	}
 
-	static requestPermission() {
+	static requestPermission(): void {
 		return; // eslint-disable-line no-useless-return
 	}
 
 	// Override default Notification permission
-	static get permission() {
+	static get permission(): 'granted' | 'denied' {
```

Comment:
We can create an ENUM type here?

Reply:
I am pretty sure I put `NotificationPermission` here 😕. Also, I don't think enum is a good idea here.